src/EntityManagerFactory.php

```diff
@@ -318,6 +320,19 @@ protected function setNamingStrategy(array $settings, Configuration $configurati
         );
     }
 
+    /**
+     * @param array         $settings
+     * @param Configuration $configuration
+     */
+    protected function setQuoteStrategy(array $settings, Configuration $configuration)
+    {
+        $strategy = Arr::get($settings, 'quote_strategy', DefaultQuoteStrategy::class);
+
+        $configuration->setQuoteStrategy(
```

Comment:
Could you instead not set quoting strategy if $strategy === null? This will keep the existing behavior and if Doctrine decides to deprecate DefaultQuoteStrategy for some reason then we do not have to do anything.